### PR TITLE
Ensure output directory exists

### PR DESCRIPTION
**Pull request by charlieDurnsfordHollands**
GitHub doesn't include empty folders, so as such a check is required to ensure that the output folder exists, either by default or where specified.

### CODE REVIEW

File: aq_zombie_finder/aq_zombie_finder.py [hunk 1]
```diff
@@ -139,19 +140,20 @@ def aq_zombie_finder():
     user = args.user
     password = args.password
     openstack_ip = args.ip
-    output = args.output
+    output_location = args.output or "output"
```
Comment (meoflynn) [inline aq_zombie_finder/aq_zombie_finder.py:143]: Here, it "output" the default string if `args.output = None`?

Comment (charlieDurnsfordHollands) [reply aq_zombie_finder/aq_zombie_finder.py]: I've moved the default to be set in the arguements to make it more readable.

**Review by meoflynn — CHANGES_REQUESTED** [review]
Looks good so far, just a couple small changes needed

**Review by DavidFair — APPROVED** [review]

**Comment by DavidFair** [comment]
@charlieDurnsfordHollands Once Megan's comments are addressed I'd be happy to merge
Also at the moment, you have this marked as a draft? If you're ready for it to be merged can you `Ready for review`?

**Review by DavidFair — APPROVED** [review]

**Pull request merged**